### DISCUSSION TITLE
Issue Loading Environment Variables with dotenv in Node.js 20.17

### DISCUSSION THREAD

**Discussion by sharmag44** [honojs/hono#3470] [Q&A]
I’m encountering an issue with loading environment variables using the dotenv package in Node.js 20.17. Despite following the typical setup for using dotenv, the environment variables defined in my .env file are not being loaded into process.env.
`import dotenv from 'dotenv';
// Load environment variables from .env file
dotenv.config({ path: '.env' });`

**Comment by safwanyp** [comment]
You do not need to use the `dotenv` package to access environment variables anymore. Although still in experimental support, Node 20 has a flag `--env-file` to specify a file with values. [Reference](https://nodejs.org/dist/latest-v20.x/docs/api/cli.html#--env-fileconfig)

As for your issue, it might help to see some more code.
